frameworks/g-cloud-9/questions/services/APIAutomationTools.yml

```diff
@@ -0,0 +1,29 @@
+name: "API automation tools"
+question: "Which standard automation tools work with your service's API?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+
+type: checkboxes
+options:
+    -
+        label: "Ansible"
+    -
+        label: "Chef"
+    -
+        label: "CloudFormation"
+    -
+        label: "OpenStack"
+    -
+        label: "SaltStack"
+    -
+        label: "Terraform"
+    -
+        label: "Puppet"
```

Comment:
Added 'List any other automation tools your service‘s API uses.' as a follow-up to this question.